uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -157,5 +204,21 @@
         }
       };
     }
+
+    /**
+     * Returns a list of mock relationship types up to the number provided.
+     *
+     * @param {Number} howMany - the number of mock relationship types to return.
+     * @return {Array}
+     */
+    function getSeveralMockRelationships (howMany) {
```

Comment:
Is there any reason we do not have/use relationships mock data?

Reply:
There are several reasons why a mock relationship file was not included:

* There are no mock relationship type files in T&W or Reqangular. In that case one would need to be created, but that's too complex for what this test is trying to accomplish.
* This Angular app purpose is to override core's Angular files. In that sense, it's not connected to Reqangular, T&W, it does not use RequireJS, etc.
* The test does not care much about the relationship type structure, just the `is_active` field. So a function was created to return a simple list of objects with the `is_active` field defined.